docs/application/web/api/6.0/device_api/mobile/tizen/nfc.html

```diff
@@ -1563,7 +1563,7 @@ <h4>Methods</h4>
 <p><span class="remark">Remark: </span>
  Security level can be determined by the manufacturer usage.
 <br>If a security issue is occurred, contact with the manufacturer of the product.
-<br>If you want to know about security level in detail, See <a href="https://www.gsma.com/digitalcommerce/wp-content/uploads/2013/12/GSMA-NFC05-NFC-Handset-APIs-Requirement-Specification-version-4-1.pdf">the GSMA specification</a>.
+<br>If you want to know about security level in detail, See <a href="https://www.gsma.com/newsroom/resources/ts26-nfc-handset-requirements/">the GSMA specification</a>.
```

Comment:
We have 3 hyperlinks to the old URL in this page, and only 1 is fixed with this update URL.

Would you please fix the other 2 hyperlinks with the old URL also?